glossary.html

```diff
@@ -33,16 +33,16 @@ <h1> Glossary </h1>
                 <dl>
                     <dt> Element </dt>
                     <dd> a tag used to represent a specific part of an HTML file </dd>
-                    <br>
+
```

Comment:
Most problems here are just validation errors that you could have caught easily by running your code through a validator.  Use the tools available to you!